process/statistics/serialize.go

```diff
@@ -6,73 +6,17 @@ import (
 	"fmt"
 
 	"github.com/ElrondNetwork/elastic-indexer-go/data"
+	logger "github.com/ElrondNetwork/elrond-go-logger"
 )
 
-// SerializeGeneralInfo will serialize statistics information
-func (sp *statisticsProcessor) SerializeStatistics(genInfo *data.TPS, shardsInfo []*data.TPS, index string) (*bytes.Buffer, error) {
-	buff, err := serializeStatisticInfo(genInfo, index)
-	if err != nil {
-		return nil, err
-	}
-
-	for _, shardInfo := range shardsInfo {
-		errSerialize := serializeShardInfo(buff, shardInfo, index)
-		if errSerialize != nil {
-			log.Warn("serializeShardInfo", "shardID", shardInfo.ShardID, "error", errSerialize)
-
-			continue
-		}
-	}
+var log = logger.GetOrCreate("process/statistics")
```

Comment:
in some places you use `logger.GetOrCreate("indexer/<path>")` and here `logger.GetOrCreate("<path>")`
please refactor in the entire project so you use one of them. I preferred the one with `indexer` preffix

Reply:
done. gj